
avoid adding -lstdc++ when linking php-common
it's needed for ext/intl
but it requires icu which already links with stdc++
so just omit PHP_ADD_LIBRARY(stdc++)

--- php-7.0.6/build/php.m4~	2016-05-16 18:31:11.000000000 +0300
+++ php-7.0.6/build/php.m4	2016-05-16 18:50:34.614991634 +0300
@@ -762,7 +762,6 @@
   if test -z "$php_cxx_done"; then
     AC_PROG_CXX
     AC_PROG_CXXCPP
-    PHP_ADD_LIBRARY(stdc++)
     php_cxx_done=yes
   fi
 ])
--- php-7.0.6/ext/intl/config.m4~	2016-04-28 21:13:06.000000000 +0300
+++ php-7.0.6/ext/intl/config.m4	2016-05-16 18:49:53.909531704 +0300
@@ -6,6 +6,7 @@
     [Enable internationalization support])])
 
 if test "$PHP_INTL" != "no"; then
+  dnl -lstdc++ not needed, will be inherited from icu
   PHP_SETUP_ICU(INTL_SHARED_LIBADD)
   PHP_SUBST(INTL_SHARED_LIBADD)
   INTL_COMMON_FLAGS="$ICU_CFLAGS -Wno-write-strings -D__STDC_LIMIT_MACROS -DZEND_ENABLE_STATIC_TSRMLS_CACHE=1"
